### ISSUE TITLE
jmc2obj 113 using more memory then 112

### ISSUE THREAD

**Issue by StandingPadAnimations** [jmc2obj/j-mc-2-obj#168]
So I've noticed that jmc2obj 113 uses more memory then 112. Even with the same world, what would have been fine to export with 112, would be a nightmare to export with 113(memory errors galore).

**Comment by mmdanggg2** [comment]
Yes it does use more memory, I will try and look for optimisations that will make this not use as much. One thing you can try is tell it to use less threads to export, this will mean less chunks are loaded at the same time. 
You should also make sure you are using a 64 bit version of java so it is able to use more than 4GB of memory.

**Comment by StandingPadAnimations** [comment]
According to my Java runtime settings, it's using x86 architecture 
![image](https://user-images.githubusercontent.com/75058058/137420313-e5fe950a-d849-472b-9ebb-be1512799c8e.png)

**Comment by mmdanggg2** [comment]
Just released 114, should need around 20% less memory now.
